src/design-system/profile-picture/image.component.tsx

```diff
@@ -20,13 +21,19 @@ export const Image = ({
   alt,
   delayMs = 600,
   selected,
+  testId = 'image',
```

Comment:
image is pretty generic, maybe we can improve it a bit by prefixing it with `profile` like `profile-image` or `profile-picture`?